### PR TITLE
[frontend] move enroll playbook action in headers

### LINKED ISSUE #9120: feat(playbook): remove the enroll-in-playbook action from the popover
## Use case

Preliminary work before deleting entity popovers from containers (Report / IR / Case RFT / Case RFI) : move the `enroll in playbook` action from outside the popover to `ContainerHeader`

## Current Workaround

Image

## If the feature request is approved, would you be willing to submit a PR?

Yes / No (Help can be provided if you need assistance submitting a PR)

### PR DESCRIPTION
**Pull request by labo-flg**
This job is part of Feature Flag `FAB_REPLACEMENT` (removal of Popover menus)

This PR is for containers Report, IR, RFT, RFI + Observables, Indicators, Artifacts.

I have also used the EE color scheme and tooltip for enroll action, as playbooks are under Entreprise Edition.

### CODE REVIEW

**Review by lndrtrbn — APPROVED** [review]

**Comment by labo-flg** [comment]
waiting to merge #8999, #9000, #9001, #8876, #8873

**Comment by labo-flg** [comment]
> waiting to merge https://github.com/OpenCTI-Platform/opencti/pull/8999, https://github.com/OpenCTI-Platform/opencti/pull/9000, https://github.com/OpenCTI-Platform/opencti/pull/9001, https://github.com/OpenCTI-Platform/opencti/pull/8876, https://github.com/OpenCTI-Platform/opencti/pull/8873

All merged.
I have updated this PR now it's ready with all screens. @lndrtrbn I'll request another review/testing please : I've added indicators, artifacts and observables.

File: opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservablePopoverFABReplaced.jsx
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservablePopoverFABReplaced.jsx:1]: So we cannot remove observables now?

File: opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableHeader.jsx [hunk 1]
```diff
@@ -49,27 +48,24 @@ const StixCyberObservableHeaderComponent = ({
       </Typography>
       <div className={classes.actions}>
         <div className={classes.actionButtons}>
-          {disableSharing !== true && (
```
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableHeader.jsx:52]: Why no more condition?

**Review by lndrtrbn — COMMENTED** [review]
Enroll button ok 👍

File: opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableHeader.jsx (hunk 1 above)
Comment (labo-flg) [reply opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableHeader.jsx:52]: this option was simply never used. We never disable sharing.

File: opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservablePopoverFABReplaced.jsx
Comment (labo-flg) [reply opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservablePopoverFABReplaced.jsx:1]: Under FF `FAB_REPLACEMENT`, the delete action is moved to the update drawer.
image

**Review by lndrtrbn — APPROVED** [review]

**Pull request merged**